lib/topic_view.rb

```diff
@@ -239,7 +239,10 @@ def page_title
       else
         title += I18n.t("inline_oneboxer.topic_page_title_post_number", post_number: @post_number)
       end
+    elsif @page > 1
```

Comment:
Can `@page` be not a number?

Reply:
It's already validated

https://github.com/discourse/discourse/blob/4a7e69d8ee8dbe3bdbe62d0c38f5892bd0fd4d19/lib/topic_view.rb#L129-L129